Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -2,158 +2,133 @@
+    2. Any project accepted for inclusion within the Project as a Sub-project must first complete, prior to becoming a Sub-project, any project contribution documentation that will assign the project name to LF Projects, LLC and other project assets to the Project.
 
-        2.  **Sub-Project Maintainers** are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a specific recognized **Sub-Project**;
+    3. The voting members of the EC consist of the Chair of the TSC of each Sub-project. 
 
-    4.  The TSC may (1) establish workflow procedures for the submission, approval, and closure/archiving of recognized **Sub-Projects** (subject to the voting rules in Section 4), (2) set ecosystem-wide minimum requirements or guidelines for the promotion of Contributors to **Sub-Project Maintainer** status (while allowing individual **Sub-Projects** to define more specific criteria), and (3) amend, adjust, refine and/or eliminate ecosystem-level roles, and create new roles, and publicly document any TSC roles and the roles and responsibilities of **Sub-Project Maintainers**, as it sees fit.
+    4. EC meetings are intended to be open to the public. The EC will designate one voting member to serve as a communication contact with LF Decentralized Trust, a directed fund of The Linux Foundation.
 
-    5.  The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by a TSC decision (as per Section 4). The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and Hyperledger Foundation, a directed fund of The Linux Foundation.
+    5. As of July 22, 2025, the initial Sub-projects of the Project and their respective Technical Steering Committees (TSCs) are those named and listed in the GOVERNANCE.MD file.
 
-    6.  **Responsibilities:** The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+3. **Technical Steering Committees**
 
-        1.  coordinating the technical direction of the Project;
+    1. A Technical Steering Committee (each, a “TSC”) will be responsible for all technical oversight of each Sub-project. Each Sub-Project has its own TSC.
 
-        2.  approving project or system proposals (including, but not limited to, archiving and changing a sub-project’s scope);
+    2. Each TSC may choose an alternative approach for determining the voting members of the applicable TSC, and any such alternative approach (together with a list of then-current TSC voting members) will be documented in the MAINTAINERS files.  TSC meetings are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
```

Comment:
This is slightly different to the current set of Fabric TSC members. Perhaps this document should coexist alongside the existing Fabric technical charter rather than replace it?